internal/juju/offers.go

```diff
@@ -47,12 +47,17 @@ type CreateOfferResponse struct {
 
 type ReadOfferInput struct {
 	OfferURL string
+	// GetModelUUID set will populate the ModelUUID field in the response.
```

Comment:
GetModelUUID set will? remove the set maybe?

Reply:
Ah yeah good catch, will do in the next PR to avoid re-running the CI.

Reply:
Sounds good